sdv/multi_table/base.py

```diff
@@ -62,20 +60,18 @@ def _initialize_models(self):
 
     def _get_pbar_args(self, **kwargs):
         """Return a dictionary with the updated keyword args for a progress bar."""
-        pbar_args = {'disable': not self.verbose}
+        pbar_args = {'disable': True}
         pbar_args.update(kwargs)
 
         return pbar_args
 
     def _print(self, text='', **kwargs):
-        if self.verbose:
-            print(text, **kwargs)  # noqa: T001
+        print(text, **kwargs)  # noqa: T001
```

Comment:
do we want to still have this only print when verbose is true? It gets called in the fit still to print an empty line

Reply:
I actually don't know. Because it verbose was True by default I kept it, but maybe we can drop it.

Reply:
We should still print the new line, but only when the progress bar is actually printed. Maybe the easier fix is to set verbose to False in the base class and set it to what the user provides in HMA.

Reply:
Alright, done in [29c5c16](https://github.com/sdv-dev/SDV/pull/1495/commits/29c5c163cea83bdd0e5fe0b550c18ed58c3f9318)